ebean-core/src/main/java/io/ebeaninternal/server/core/DefaultBeanLoader.java

```diff
@@ -87,26 +92,12 @@ private void loadManyInternal(EntityBean parentBean, String propertyName, boolea
       query.setLoadDescription("lazy", null);
     }
 
-    query.select(parentDesc.idBinder().idSelect());
-    if (onlyIds) {
-      query.fetch(many.name(), many.targetIdProperty());
-    } else {
-      query.fetch(many.name());
-    }
-    if (filterMany != null) {
-      query.setFilterMany(many.name(), filterMany);
-    }
-
-    query.where().idEq(parentId);
-    query.setBeanCacheMode(CacheMode.OFF);
-    query.setMode(Mode.LAZYLOAD_MANY);
-    query.setLazyLoadManyPath(many.name());
```

Comment:
CHECKME: what do we need here? 
- I think we've lost the `CacheMode.OFF` for refresh? - see: https://github.com/ebean-orm/ebean/pull/3486/files#diff-5ce08d7970ceac437b60d46bbe9a40b0a4893f2211045684f0c29342c3322541R105
- Do we need `Mode.LAZYLOAD_MANY`